shells/web-shell/elements/web-config.js

```diff
@@ -42,7 +42,8 @@ const configOptions = {
   },
   arc: {
     aliases: ['arckey'],
-    persistToUrl: true
+    // TODO(sjmiles): disable for now as persistence is defeated for StorageNG
```

Comment:
This prevents `arc=....` from appearing in the URL bar. We had to defeat much of the persona-persistence features because StorageNG doesn't yet support SyntheticStores. Although, on reflection I think arcs _arc_ still technically persisted (there just is no API support for discovering them ex post facto). Maybe I should put this back.